src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -1487,7 +1487,7 @@ void GrpcLb::OnBalancerChannelConnectivityChangedLocked(void* arg,
         grpclb_policy->lb_call_backoff_.Reset();
         grpclb_policy->StartBalancerCallLocked();
       }
-      // Fall through.
```

Comment:
replace `// Fall through.`  by `/* fallthrough */` which should silence the warning (if any)?